doc/src/manual/metaprogramming.md

```diff
@@ -927,27 +932,33 @@ mutable state or cause any mutation of global state, we see the following behavi
 Observe that the generated function *cannot* call any method that was not defined
 prior to the *definition* of the generated function itself.
 
-```julia
-julia> # initially f(x) has one definition:
+Initially `f(x)` has one definition
 
+```jldoctest redefinition
 julia> f(x) = "original definition";
+```
 
-julia> # start some other operations that use f(x):
+Start some other operations that use f(x):
```

Comment:
As a standalone sentence this doesn't work very well. I'd say something like "define other operations that use `f(x)`"